<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QQuickWidget" name="quickWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>731</height>
     </rect>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::SizeRootObjectToView</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>691</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_presupuesto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Presupuesto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_resumen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Resumen / Consulta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_factura">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Añadir Facutra</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>9</y>
      <width>921</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="presupuesto">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre Empresa</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre Presupuesto</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>20</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Fecha</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_empresa">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_nombre_presupuesto">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="input_fecha">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>50</y>
        <width>110</width>
        <height>26</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhDate|Qt::ImhPreferNumbers|Qt::ImhSensitiveData</set>
      </property>
     </widget>
     <widget class="QTabWidget" name="first_tab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>901</width>
        <height>351</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Etapas">
       <property name="accessibleName">
        <string>Etapas</string>
       </property>
       <property name="accessibleDescription">
        <string/>
       </property>
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QTextEdit" name="input_descripcion">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>541</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Descripción</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>10</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Cantidad</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="input_cantidad">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>40</y>
          <width>44</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>10</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>U/M</string>
        </property>
       </widget>
       <widget class="QComboBox" name="sel_unidad">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>40</y>
          <width>71</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>10</y>
          <width>66</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Precio</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_precio">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>40</y>
          <width>81</width>
          <height>25</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers|Qt::ImhSensitiveData</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>10</y>
          <width>66</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>% Desc</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_descuento">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>40</y>
          <width>71</width>
          <height>25</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers|Qt::ImhSensitiveData</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>260</y>
          <width>66</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Importe</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_importe">
        <property name="geometry">
         <rect>
          <x>732</x>
          <y>260</y>
          <width>161</width>
          <height>25</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>480</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>SubTotal</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>520</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>IVA</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_iva">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>520</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>560</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Total</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_total">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>560</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_guardar">
      <property name="geometry">
       <rect>
        <x>427</x>
        <y>594</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_buscar_empresa">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_add_tab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>484</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Añadir Etapa</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_subtotal">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>480</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="resumen">
     <widget class="QLineEdit" name="input_search_name">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>291</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>          Nombre Empresa ó Presupuesto</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="input_search_date">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>0</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_search">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>0</y>
        <width>88</width>
        <height>25</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>591</width>
        <height>611</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="factura">
     <widget class="QDial" name="dial">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>190</y>
        <width>50</width>
        <height>64</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1126</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
